classes/class-wc-connect-privacy.php

```diff
@@ -46,6 +47,27 @@ public function add_privacy_message() {
 		wp_add_privacy_policy_content( $title, $content );
 	}
 
+	/**
+	 * If WooCommerce order data erasure is enabled, display a warning on the erasure page
+	 */
+	public function add_erasure_notice() {
+		global $pagenow;
+		if ( ( $pagenow !== 'tools.php' ) || ( $_GET['page'] !== 'remove_personal_data' ) ) {
```

Comment:
+1 to using `get_current_screen()`.

This will also need an `isset()` check on `$_GET['page']`.